site/themes/vscode/README.md

```diff
@@ -0,0 +1,12 @@
+![Theme Screenshot](https://raw.githubusercontent.com/deislabs/cnab.io/master/themes/cnab/img/screenshot.png?token=AAp4ctyXT6Xuaer1Vm8SXT0HscNhS5WGks5cBvT9wA%3D%3D)
+
+CNAB Theme
```

Comment:
Should we use a different name here, or is this the name of the theme?  I'm guessing it's not very visible so won't cause confusion.